### ISSUE TITLE
MBIS fails via QCEngine

### ISSUE THREAD

**Issue by jthorton** [psi4/psi4#2272]
Using psi4-1.4rc1, I was able to compute `MBIS_CHARGES` via QCEngine however after PR #2127 this now fails with the following error message. While this definitely needs fixing I think it might also be nice to have the option to turn off this feature and only calculate the ratio if passed as an scf property?
```
RuntimeError: 
Fatal Error: Wavefunction::scalar_variable: Requested variable MBIS FREE ATOM O VOLUME was not set!
Error occurred in file: /Users/github/builds/conda-builds/psi4-multiout_1628080613326/work/psi4/src/psi4/libmints/wavefunction.cc on line: 1335
```

MWE 
```python
import qcelemental as qcel
from qcelemental.models import molecule
import qcengine as qcng

mol = qcel.models.Molecule.from_data("""
     O  0.0  0.000  -0.129
     H  0.0 -1.494  1.027
     H  0.0  1.494  1.027
     """)

# build the mbis task
model = qcel.models.AtomicInput(
    molecule=mol,
    driver="energy",
    model={"method": "SCF", "basis": "sto-3g"},
    keywords={"scf_properties": ["MBIS_CHARGES"]}
)

result = qcng.compute(model, "psi4")

print(result.dict())
```
versions
psi4-1.4+9485035 
qcengine-0.19.0
qcelemental -0.21.0

**Comment by loriab** [comment]
Apparently very easy to fall into endless recursion with MBIS volumes due to some necessary inputs also calculated by `oeprop()` and involving calls to `energy()`.

You're interested in MBIS CHARGES as standalone (w/o volumes and extents) to avoid the cost of all the atomic calcs since you don't need the volume?

**Comment by jthorton** [comment]
Thanks for the quick response,  that makes sense!

>You're interested in MBIS CHARGES as standalone (w/o volumes and extents) to avoid the cost of all the atomic calcs since you don't need the volume?

Sort of the only bit I need is the charges and volumes but not the volume ratio so not the free atom calculations added in #2127. So basically the performance before that PR my idea would be that the ratios are only calculated if something like `MBIS RATIOS` is passed which like you say would also avoid the cost of the extra calculations.

**Comment by djcole56** [comment]
Hi @loriab, I was just wondering if there are any updates on this issue please? I'm not sure how easy it would be to revert to behaviour before PR #2127? - at this point we had fully functioning access to both MBIS charges and atomic volumes. The calculation of the free atom volumes does not seem like a key feature to me (they can be easily tabulated). Thanks for your help.

**Comment by loriab** [comment]
Hi @djcole56, the immediate issue is fixed in #2273, and I think that can be merged after a couple test case adjustments. We were going to backport that one.

The other half of the issue that isn't troubling you @jeffschriber was going to work on.

Are you more interested in the v1.4.x or the v1.5.devx series?

**Comment by djcole56** [comment]
Great, thanks @loriab. We can work with either, but if you can get it in the v1.4.x series I think that would help us out a lot for ensuring our current application of this functionality is reproducible going forward.

**Comment by loriab** [comment]
closed by #2273 and the release of https://github.com/psi4/psi4/releases/tag/v1.4.1
